ios/UnflowMetadataBridge.swift

```diff
@@ -20,11 +20,20 @@ struct UnflowMetadataBridge {
         if let analyticsValue = value as? UnflowAnalyticsValue {
             return analyticsValue
         } else if let numberValue = value as? NSNumber {
-            return numberValue.doubleValue
+            if CFGetTypeID(numberValue) == CFBooleanGetTypeID() {
+                return numberValue.boolValue
+            } else {
+                return numberValue.doubleValue
+            }
         } else if let stringValue = value as? NSString {
-            return String(stringValue)
-        } else if let date = value as? NSDate {
-            return date as Date
+            let swiftString = String(stringValue)
```

Comment:
@SwiftyAlex I decided to actually write a parser to keep all the React weirdness in the same place. Could probably be more idiomatic if you care to take a look